es-ds-components/components/es-carousel.vue

```diff
@@ -1,40 +1,59 @@
 <script setup lang="ts">
 /*
     TODO:
-     - responsive carousels briefly show mobile amount of dots on desktop, until hydration
-        - i think we may be able to fix this with CSS, because we know how many dots there should be
-        - (something PrimeVue Carousel should really be doing)
      - circular has quirky behavior when numVisible doesn't match numScroll
         - you can see this in the circular autoplay example
         - i'm not sure if this is fixable
+     - figure out peek behavior
+     - prop to position the arrows at the bottom two corners of a full-width slide, like homepage
```

Comment:
Potential TODO: the prev arrow in the circular example is disabled until hydration

Reply:
Good catch! Fixed.